src/plugins/dynamic_dagrun_operator.py

```diff
@@ -1,10 +1,8 @@
-from __future__ import annotations
-
 from datetime import datetime
-from typing import Any, Optional
+from typing import Any, Dict, Optional
 
 from airflow.models.dag import DagModel, DagRun
-from airflow.operators.dagrun_operator import TriggerDagRunOperator
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
```

Comment:
heeft pyupgrade dit ook gedaan?

Reply:
Nope, that was me. For some strange reason it changed `Optional[datetime]` into `datetime | None` for this file even though I ran it with `--py38-plus`. So I had to revert it manually. While doing so I saw the deprecated import and fixed that immediately. Should have been a separated commit.

Reply:
Done.